### PR TITLE
Optie toegevoegd om de adviesraad van het KB ook te kunnen mailen

### PR DESCRIPTION
**Pull request by HugoPeters1024**
Slechts een functie toegevoegd.

### CODE REVIEW

File: Google Scripts/Projects/Notulen/Notulen.js [hunk 1]
```diff
@@ -47,79 +48,98 @@ function placeAPonTop () {
...
+      //"idMembers": ,//(optional)Valid Values: A comma-separated list of objectIds, 24-character hex strings
+      //"idCardSource": ,//(optional)Valid Values: The id of the card to copy into a new card.
+      //"keepFromSource": ,//(optional)Default: all Valid Values: Properties of the card to copy over from the source.
+    };
+
+    // Because payload is a JavaScript object, it will be interpreted as
+    // an HTML form. (We do not need to specify contentType; it will
+    // automatically default to either 'application/x-www-form-urlencoded'
+    // or 'multipart/form-data')
+    var _url = 'https://api.trello.com/1/cards?key=' + api_key + '&token=' + api_token; //optional... -&cards=open&lists=open'
```
Comment (yorickvanzweeden) [inline Google Scripts/Projects/Notulen/Notulen.js]: @HugoPeters1024 Url_base 'https://api.trello.com/1/' is also declared as variable in line 17/18

File: Google Scripts/Projects/Notulen/Notulen.js [hunk 2]
```diff
@@ -47,79 +48,98 @@ function placeAPonTop () {
...
+      due = "";
+
+    //POST [/1/cards], Required permissions: write
+    var payload = {
+      "name": AP.action, //(required) Valid Values: a string with a length from 1 to 16384
+      "desc": AP.desc, //(optional)Valid Values: a string with a length from 0 to 16384
+      "pos": "top", //(optional) Default: bottom Valid Values: A position. top, bottom, or a positive number.
+      "due": due, //(required) Valid Values: A date, or null
+      "idList": listId, //(required)Valid Values: id of the list that the card should be added to
+      //"labels": ,//(optional)
```
Comment (yorickvanzweeden) [inline Google Scripts/Projects/Notulen/Notulen.js]: @HugoPeters1024 If you are not going to use these, why include them? It's better to refer to the original documentation than to include it yourself.

File: Google Scripts/Projects/Notulen/Notulen.js [hunk 3]
```diff
@@ -47,79 +48,98 @@ function placeAPonTop () {
...
+      //"idMembers": ,//(optional)Valid Values: A comma-separated list of objectIds, 24-character hex strings
+      //"idCardSource": ,//(optional)Valid Values: The id of the card to copy into a new card.
+      //"keepFromSource": ,//(optional)Default: all Valid Values: Properties of the card to copy over from the source.
+    };
+
+    // Because payload is a JavaScript object, it will be interpreted as
```
Comment (yorickvanzweeden) [inline Google Scripts/Projects/Notulen/Notulen.js]: @HugoPeters1024 Again, I would prefer to refer to the original documentation.

File: Google Scripts/Projects/Notulen/Notulen.js (hunk 2 above)
Comment (HugoPeters1024) [inline Google Scripts/Projects/Notulen/Notulen.js]: seemed useful for quick updates in the future and is far more clear than the original documentation.

**Pull request closed without merging**